samples/zigbee/ncp/src/main.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file
+ *  @brief Zigbee Network Co-processor sample
+ */
+
+#include <drivers/uart.h>
+#include <usb/usb_device.h>
+#include <logging/log.h>
+#include <zb_nrf_platform.h>
+#include <zb_osif_ext.h>
+
+LOG_MODULE_REGISTER(app);
+
+int main(void)
+{
+	LOG_INF("Starting Zigbee Network Co-processor sample");
+
+#ifdef CONFIG_USB
```

Comment:
Is this correct. What if I just use UART but not USB?

Reply:
This part is only needed for USB. UART transport is ok without it